src/ucp/core/ucp_am.c

```diff
@@ -567,7 +567,7 @@ ucp_am_send_short(ucp_ep_h ep, uint16_t id, uint16_t flags, const void *header,
     return uct_ep_am_short_iov(ucp_ep_get_am_uct_ep(ep), am_id, iov, iov_cnt);
 }
 
-static ucs_status_t ucp_am_contig_short(uct_pending_req_t *self)
+ucs_status_t ucp_am_contig_short(uct_pending_req_t *self)
```

Comment:
not needed

Reply:
it's needed for `src/ucp/wireup/wireup.c`